src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  max-width: $calendarWidth;
+  padding: 50px $hPadding;
+
+  &__day {
+    @include day($cellSize);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    background-color: #eee;
```

Comment:
As for me, it's better to put all the colors in a variable

Reply:
> As for me, it's better to put all the colors in a variable

Yeah, for sure, but we use each color only once so I see no need of doing that in this task